Tools/validate_yaml.py

```diff
@@ -1,68 +1,58 @@
 #! /usr/bin/env python3
 """ Script to validate YAML file(s) against a YAML schema file """
 
-from __future__ import print_function
-
 import argparse
-import sys
-
-try:
-    import yaml
-except ImportError as e:
-    print("Failed to import yaml: " + str(e))
-    print("")
-    print("You may need to install it using:")
-    print("    pip3 install --user pyyaml")
-    print("")
-    sys.exit(1)
-
-try:
-    import cerberus
-except ImportError as e:
-    print("Failed to import cerberus: " + str(e))
-    print("")
-    print("You may need to install it using:")
-    print("    pip3 install --user cerberus")
-    print("")
-    sys.exit(1)
```

Comment:
Well, to be honest I find them to be a bit clunky and not super useful. The import statement raises an error if it can't find it and that's pretty readable. Also, pip3 is not the only way to install a package, i.e. system package managers. I feel like its better to just maintain a requirements.txt in the tree that we point to in documentation. What do you think?

@dagar thoughts?

Reply:
I know, advanced users will know what to do. We added them when adding new external requirements, for people who updated their branch and it didn't compile anymore.